tests/unit/test_core/test_pacing.py

```diff
@@ -0,0 +1,134 @@
+"""Tests kytos.core.pacing module."""
+
+import asyncio
+import time
+
+import pytest
+
+from kytos.core.pacing import Pacer, PacerWrapper
+
+
+class TestPacer:
+    """Pacer tests"""
+
+    @pytest.fixture
+    def pacer(self) -> Pacer:
+        """Setup the pacer"""
+        return Pacer("memory://")
+
+    @pytest.fixture(
+        params=[
+            'fixed_window',
+            pytest.param(
+                'elastic_window',
+                marks=pytest.mark.skip(reason='Inconsistent behaviour')
```

Comment:
David, good intention to skip, but since we won't support this in the short/medium term, let's nuke this arg here, and then if one day this needs to be supported then a developer can add again this fixture param and continue from here. See what I mean? Otherwise, these tests cases will be keeping skipped for so long that's not worth it, so basically only noise in the test reports (for something that we won't support soon).